python/humbug/report.py

```diff
@@ -44,6 +45,7 @@ def __init__(
         self.bugout_token = bugout_token
         self.bugout_journal_id = bugout_journal_id
         self.timeout_seconds = timeout_seconds
+        self.additional_tags = tags
 
     def system_tags(self) -> List[str]:
         tags = [
```

Comment:
I don't want to store `client_id:{self.client_id}` and `session_id:{self.session_id}` in `self.additional_tags`. We should populate them into `tags` array separately.

Add lines like:
```python
def session_tags(self):
    return ["client_id:{}".format(self.client_id), "session_id:{}".format(self.session_id)]
```

```
report.tags += self.session_tags()
```

Reply:
As I understand it is not actual any more, after v0.1.3 was released